manifests/elasticsearch-operator.package.yaml

```diff
@@ -2,4 +2,4 @@
 packageName: elasticsearch-operator
 channels:
 - name: preview
-  currentCSV: elasticsearch-operator.v4.2.0
+  currentCSV: elasticsearch-operator.v4.1.1
```

Comment:
This would be a more general question...  Now we have these 4.x branches - [1].  I'm not sure how this currentCSV is being used, but is it not necessary for the operator version and the branch to be in sync?  Like if the branch is master, pointing v4.3.0 (only if exists???); if release-4.2, v4.2.0; if release-4.1, v4.1.x (x is the latest)?  (Or it does not matter at all?)

[1]
  origin/release-4.0
  origin/release-4.1
  origin/release-4.2
  origin/release-4.3

Reply:
this is what OLM would consume. This is going to get cherry picked into `release-4.1` which is why i have it pointing to the `4.1.1` csv at this time.